src/renderer/pages/Staking/Operations/ChangeValidators/ChangeValidators.tsx

```diff
@@ -192,6 +161,7 @@ export const ChangeValidators = () => {
   };
 
   const explorersProps = { explorers, addressPrefix, asset };
+  const multisigTx = isMultisig(txAccounts[0]) ? wrapTx(txs[0], api, addressPrefix) : undefined;
```

Comment:
Why do we still have `isMultisig` in concrete flows? The primary goal was to abstract it away

Reply:
Agree here. As I understand it must be in the OperationHeader. The idea was that if an operation has a wrapper (such as multisig) then it must contain OperationHeader in the tree that wraps the final transaction object to introduce additional logic to transaction. Here as we can see we do this in the ui for the call logic.